Makefile

```diff
@@ -26,10 +31,6 @@ USEMODULE += gnrc_ipv6_default
 USEMODULE += gnrc_udp
```

Comment:
this USERMODULE can be removed

Reply:
Why? It's used by `gnrc_udp_sock`

Reply:
I am not sure about it, because when I did the test without that module, the result is the same, thats mean its no necessarr

Reply:
Resolved in #59